### PR TITLE
Add metabase docker container

### PR DESCRIPTION
**Pull request by ThrowsException**
Adds local metabase container #238

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +76,20 @@ This project implements the [Scripts To Rule Them All](https://github.com/github
...
+
+The Metabase containers can be configured to the postgres database by using there settings aftering creating the containers. 
+| Setting           | Value             |
+| ----------------- | ----------------- |
+| Name              | (Your preference) |
+| Host              | db                |
+| Port              | 5432 (default)    |
+| Database name     | postgres          |
+| Database username | (leave blank)     |
+| Database passowrd | (leave blank)     |
```
Comment (themightychris) [inline README.md]: Can this config get passed into the metabase container via env from the docker-compose file?

Comment (ThrowsException) [reply README.md]: I did look around their docs I didn't see a way to have a default config with either a mounted config file or env variables. The only other thing I could think of is setting up an initial connection and checking in the `.db` files it creates and using some scripting up some kind of startup but I wasn't sure if that was worth the effort for this

**Comment by mariekers** [comment]
@MooseQuest this was to make it so the app and metabase were spun up at the same time

**Comment by keegan2149** [comment]
I’m not getting an error message during the docket compose. I can just add the environment vars for Postgres. I feel like I’m missing something though.

**Comment by MooseQuest** [comment]
Lets setup sometime to review this,maybe tomorrow?

**Pull request closed without merging**